sky/adaptors/common.py

```diff
@@ -1,24 +1,47 @@
 """Lazy import for modules to avoid import error when not used."""
+import importlib
 from typing import Optional
 
 
 class LazyImport:
-    """Lazy importer for heavy modules."""
+    """Lazy importer for heavy modules or cloud modules only when enabled.
 
-    def __init__(self, module_name, import_error_message: Optional[str] = None):
-        self.module_name = module_name
-        self.module = None
+    We use this for pandas and networkx in SkyPilot module, as they can be time-
+    consuming to import (0.1-0.2 seconds). With this module, we can avoid the
+    unnecessary import time when the module is not used.
```

Comment:
```suggestion
    We use this for pandas and networkx, as they can be time-
    consuming to import (0.1-0.2 seconds). With this class, we can avoid the
    unnecessary import time when the module is not used.
```
Could we explicitly give an example for "when the module is not used (e.g., ...)"? I'm wondering if it refers to `sky launch / exec --cloud kubernetes` or something else?

Reply:
Good point! Added examples.